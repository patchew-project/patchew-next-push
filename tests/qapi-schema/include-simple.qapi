{ 'include': 'include-simple-sub.qapi' }
