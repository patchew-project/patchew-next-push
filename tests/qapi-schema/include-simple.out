object q_empty
enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
module include-simple.qapi
include include-simple-sub.qapi
module include-simple-sub.qapi
enum Status ['good', 'bad', 'ugly']
